data/categories.txt

```diff
@@ -10622,6 +10665,7 @@ it:Attrezzatura sportiva|trekking|arrampicata|campeggio
 ja:アウトドア用品|トレッキング|クライミング|店
 ko:아웃도어 장비|야외 활동 장비|산행 장비|등산 장비|등산 물품|산행 물품
 nb:Fritidsutstyr|vandring|klatring|camping
+nl:4Outdooruitrusting|klimmen|kamperen|kampeer|buiten|wandel|klim
```

Comment:
I'm sorry the button doesn't appear

nl:4Outdooruitrusting|buitenwinkel|buitensportwinkel|kampeerwinkel|klimmen|kamperen|kampeer|buiten|wandel|klim